docs/getting_support.rst

```diff
@@ -3,45 +3,435 @@
+your public key. Paste your armored public key into the empty text box
+on the left and hit **Save**.
+
+|PGPSave|
+
+Importing the server's public key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now, import the public key for our Redmine server into your local GPG
+keyring. It is available on the right hand side of the page in the
+column with the heading "Redmine Server (support@freedom.press)", in the
+text box labeled "Public PGP key".
+
+Select the entire public key and copy it to your clipboard. Import the
+copied public key into your local GPG keyring.
+
+You can `download the key <https://freedom.press/documents/37/redmine_public_key.asc>`_ 
+or fetch it from a `keyserver <https://keys.openpgp.org/vks/v1/by-fingerprint/D0E0B2F2B71BA4E48278037D9EA33029E9FBBA2E>`_ (fingerprint:
+``D0E0B2F2B71BA4E48278037D9EA33029E9FBBA2E``).
+
+
+.. todo:: Add section on testing encrypted email after setting it up
+
+.. |PGPMenu| image:: images/support/pgp_menu.png
+.. |PGPUpload| image:: images/support/pgp_upload.png
+.. |PGPSave| image:: images/support/pgp_save.png
+
+Creating a new issue via email
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sending an email to support@freedom.press will automatically create a
+new issue in your project, using the subject line of the email for the
+Subject and the body of the email for the Description.
+
+Always sign and encrypt your emails
+-----------------------------------
+
+To maintain the security of conversations around support requests, you
+should always sign and encrypt your email to support@freedom.press if
+you have a Redmine account.
+
+Our server is configured to reject emails that do not have a valid
+signature. We cannot reject unencrypted emails due to a limitation in
+the Redmine-OpenPGP plugin we are using, but we are logging them and
+will harangue you if you send us unencrypted email.
+
+PGP/MIME vs. inline PGP
 -----------------------
+
+When you're replying to Redmine via email, you **must use PGP/MIME**
+instead of inline PGP. The default settings for the Enigmail plugin in
+Thunderbird should work.
+
+Check your mail client and GPG integration tool's settings to ensure you
+are using PGP/MIME. One notable exception to this is Mailvelope, which
+cannot send PGP/MIME (you shouldn't be using Mailvelope anyway).
+
+Per-recipient Rules
+-------------------
+
+You may not want to use these settings (PGP/MIME, always sign) for all
+of your email. If not, most email clients support the concept of
+"Per-recipient rules", which allow you to configure specific settings on
+a per-recipient basis.
+
+Below is an example of how to configure a per-recipient rule for
+support@freedom.press in Thunderbird/Enigmail. If you are not using
+Thunderbird/Enigmail, consult your mail client's documentation.
+
+Thunderbird
+-----------
+
+To check your global settings, go to Edit > Account Settings in
+Thunderbird. Navigate to the "OpenPGP Security" tab of the email account
```

Comment:
The menu item is now called "End-to-End Encryption".

AFAICT there's no longer a PGP/MIME option, I think it's the only thing that's supported?

And...I can't figure out how to set a per-recipient rule. If I compose an email to support@ it does prompt me that encryption is possible, but I haven't yet found an option to enforce it.

Reply:
Given these changes to Thunderbird (and the fact that the options we're flagging are either removed/non-existent), and that we don't provide these details for any other email clients, I'm going to suggest that we remove this Thunderbird-specific section entirely.

Reply:
Yeah I think that's reasonable. If anything, we could just link to https://support.mozilla.org/en-US/kb/openpgp-thunderbird-howto-and-faq and leave it at that.

Now that we're discussing it, I am curious 1) how many people are using PGP email for redmine and 2) what email clients they're using. The former seems like something we could get out of mail server logs or the redmine DB (checking if people have a key set), the latter would require logging incoming mail before redmine processed it? AFAICT Thunderbird sends the `User-Agent` field in cleartext, idk about other clients. Maybe not worth the effort, but I'm at least curious :)